src/Layout/Presentation/SingleDistrict/InfoBox.tsx

```diff
@@ -16,7 +19,7 @@ export class InfoBox extends React.Component<InfoBoxProps> {
                       " hadde nærmeste kvotient og trengte ",
                       vulnerable.moreVotesToWin,
                       " flere stemmer for å vinne mandatet. ",
-                          vulnerableVotes.partyCode,
+                          <b key="vulnerableVotes">{partyMap[vulnerableVotes.partyCode]}</b>,
```

Comment:
There are a lot of &lt;b&gt;'s here. It really would be a lot better to have a class, as it might not specify the behaviour we want if it is to be changed at some point in the future (whereas a class would represent the intention/what should be affected).